CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.24.0...master
 
+### Added
+- Add `nodeId` to `PerformanceServerTiming`. Aid with [JPERF-1409].
```

Comment:
Could we make this a lib's internal contract and not expose it in `PerformanceServerTiming`? We'd expose it only in `BackendTimeSlot`.

Reply:
I'll remove the javascript detail from changelog